clang/lib/Analysis/FlowSensitive/HTMLLogger.css

```diff
@@ -122,10 +122,22 @@ code.line:has(.bb-select):before {
   font-size: x-small;
   flex-grow: 1;
 }
-.value summary {
+.value > summary {
```

Comment:
Figured I would change this too for consistency. (And maybe it's also more efficient?)